packages/common/src/store/ui/stripe-modal/sagas.ts

```diff
@@ -139,7 +139,7 @@ function* handleCancelStripeOnramp() {
   yield* put(setVisibility({ modal: 'StripeOnRamp', visible: 'closing' }))
 
   if (onrampCanceled) {
-    yield* put(onrampCanceled)
+    yield* put({ ...onrampCanceled })
```

Comment:
I don't see where we are using the failed action and passing along the error?

Reply:
The failed one already looks like this.